provider/cmd/pulumi-resource-azure/schema.json

```diff
@@ -100547,6 +101272,8 @@
                 },
                 "type": "object",
                 "required": [
+                    "identities",
+                    "location",
```

Comment:
No action needed here - but it is interesting to observe that new required fields would in a traditional sense also be a breaking change - but seems to not be considered one here by the upstream provider?